data/questions.js

```diff
@@ -3169,1336 +3163,1216 @@ export const questions = {
+                },
+                {
+                    "text": "North Carolina",
+                    "idCode": "36"
+                },
+                {
+                    "text": "North Dakota",
+                    "idCode": "37"
+                },
+                {
+                    "text": "Ohio",
+                    "idCode": "38"
+                },
+                {
+                    "text": "Oklahoma",
+                    "idCode": "39"
+                },
+                {
+                    "text": "Oregon",
+                    "idCode": "40"
+                },
+                {
+                    "text": "Pennsylvania",
+                    "idCode": "41"
+                },
+                {
+                    "text": "Puerto Rico",
+                    "idCode": "42"
+                },
+                {
+                    "text": "Rhode Island",
+                    "idCode": "43"
+                },
+                {
+                    "text": "South Carolina",
+                    "idCode": "44"
+                },
+                {
+                    "text": "South Dakota",
+                    "idCode": "45"
+                },
+                {
+                    "text": "Tennessee",
+                    "idCode": "46"
+                },
+                {
+                    "text": "Texas",
+                    "idCode": "47"
+                },
+                {
+                    "text": "United States Virgin Islands",
+                    "idCode": "48"
+                },
+                {
+                    "text": "Utah",
+                    "idCode": "49"
+                },
+                {
+                    "text": "Vermont",
+                    "idCode": "50"
+                },
+                {
+                    "text": "Virginia",
+                    "idCode": "51"
+                },
+                {
+                    "text": "Washington",
+                    "idCode": "52"
+                },
+                {
+                    "text": "West Virginia",
+                    "idCode": "53"
+                },
+                {
+                    "text": "Wisconsin",
+                    "idCode": "54"
+                },
+                {
+                    "text": "Wyoming",
+                    "idCode": "55"
                 }
             ]
         },
         {
-            "numOptionsAllowed": "0.0",
-            "question": "First Name of Person Involved",
-            "id": "xkXuEz84",
-            "answerType": "openTextBox",
+            "numOptionsAllowed": "1.0",
+            "question": "Crash County",
+            "id": "8V2H4hT4",
+            "answerType": "advancedDropDown",
             "display": [
-                "passenger"
+                "road"
             ],
-            "humanReadableId": "passenger-0"
+            "humanReadableId": "road-11",
+            "autoMethod": "county"
```

Comment:
i think we still need to add a "answerOptions" field with an empty array [] for now just like the weather conditions advancedDropDown as currently the logic in line #42 of AdvancedDropDown.js is giving an error when trying to load Road (error: undefined is not an object (evaluating 'data.answerOptions.length')). Is it possible to also add a helper text for County saying "Choose a state first"? Thanks so much!